app/assets/javascripts/protocols_show.js.coffee

```diff
@@ -4,14 +4,11 @@ $ ->
 
     # Bootstrap Tab persistence
 
```

Comment:
app/views/protocols/show.html.haml has conditions in the partial for that (line 4 e.g.)